amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -0,0 +1,183 @@
+= Amazon S3 Connector 6.0 Additional Configuration
+:page-aliases: connectors::amazon/amazon-s3-connector-config-topics.adoc
+
+== Create Object Operation
```

Comment:
Can be dropped now (the description is at the field level, and there is no such specific thing).